modules/ROOT/partials/misc/admon-iframe-only.adoc

```diff
@@ -0,0 +1 @@
+NOTE: This option is only supported when {productname} is run in _iframe_ mode. It is not supported in _inline_ mode. For more information on the differences between the editing modes, see xref:use-tinymce-classic.adoc[Classic editing mode].
```

Comment:
This shouldn't say `iframe` mode... the name is `classic` mode, as noted by the last part of this admonition.